jinahub/indexers/searcher/HnswlibSearcher/hnswlib_searcher.py

```diff
@@ -20,20 +20,21 @@ class HnswlibSearcher(Executor):
     def __init__(
         self,
         limit: int = 10,
-        metric: str = 'cosine',
+        metric: str = 'euclidean',
```

Comment:
Let's not change the default

Reply:
This revision is to make all of the indexers (simple, faiss, annoy, hnsw) use the same default metric.

Reply:
@tadejsv Apart from the compatibility, any other reason to reject to change the default?